### DISCUSSION TITLE
SaltEdge data import not getting latest transactions

### DISCUSSION THREAD

**Discussion by clintonboys** [firefly-iii/firefly-iii#6164] [Firefly III]
Hi - I am running Firefly and the data importer through Docker (specifically Portainer on a Sinology NAS), it's working great except for when I try to update the data: in the logs I can see that the Data Importer is accessing Salt Edge with the correct connection ID and account ID, but the list of transactions it receives stops on May 28, whereas in the Salt Edge browser UI I can see many more recent transactions (see images). 

Perhaps there is some cache somewhere I need to clear?

<img width="1369" alt="Screen Shot 2022-06-14 at 15 31 23" src="https://user-images.githubusercontent.com/8884079/173578223-27574cc0-3073-46b8-87fd-b31a98001bac.png">

<img width="1369" alt="Screen Shot 2022-06-14 at 15 31 46" src="https://user-images.githubusercontent.com/8884079/173578217-ac757ee8-42ee-4e7a-be29-2339df23707e.png">

<img width="686" alt="Screen Shot 2022-06-14 at 15 32 33" src="https://user-images.githubusercontent.com/8884079/173578212-c687d99c-d922-4f55-8515-c68ad98d8e6d.png">

**Comment by JC5** [comment]
There's no cache on my end. You may have to recreate the connection.

**Reply by clintonboys** [reply]
Hi, thanks for replying, yes I tried that, I am still not receiving all the transactions even with a brand new connection.

**Reply by JC5** [reply]
Mmm, this is me running out of ideas. I know some banks cache the results but you're obviously getting the latest ones. Just not through the API. Is there a date range in your config perhaps?

**Reply by clintonboys** [reply]
I don't believe so; this is the config (downloaded just now before trying the import again):

```
{
    "version": 3,
    "source": "fidi-0.9.9",
    "created_at": "2022-06-15T09:38:05+00:00",
    "date": "",
    "default_account": 52,
    "delimiter": "comma",
    "headers": false,
    "rules": true,
    "skip_form": false,
    "add_import_tag": true,
    "roles": [],
    "do_mapping": [],
    "mapping": [],
    "duplicate_detection_method": "classic",
    "ignore_duplicate_lines": false,
    "unique_column_index": 0,
    "unique_column_type": "external-id",
    "flow": "spectre",
    "identifier": "zzz",
    "connection": "yyy",
    "ignore_spectre_categories": false,
    "map_all_data": false,
    "accounts": {
        "xxx": 52
    },
    "date_range": "all",
    "date_range_number": 30,
    "date_range_unit": "d",
    "date_not_before": "",
    "date_not_after": "",
    "nordigen_country": "",
    "nordigen_bank": "",
    "nordigen_requisitions": [],
    "nordigen_max_days": "90",
    "conversion": false,
    "ignore_duplicate_transactions": true
}
```

**Comment by JC5** [comment]
I'm sorry, I'm not sure what's going on. On my end, the data is refreshed every time. Perhaps this is something that Salt Edge support can help with? I know for my bank, transactions usually lag a day but it's at least consistent.
